ONSdigital/eq-questionnaire-runner
# Add mobile number validation

### What is the context of this PR?

Updates runner to introduce a new answer type `MobileNumber` which validates mobile numbers based on the requirements of the card (copied in below for reference):

***Validation:***
- Needs to ignore spaces
- Remove leading: 044, +44, 0 
- Remove all: Brackets   - all types, Hyphen, Full Stop, Comma, Whitespace, TAB space, Forward slash

- Check for numbers only - 'Valid UK number' error if not
- Check if 10 characters- 'Valid UK number' error if not

Replay the number they have entered in the way it has been entered

***Error messages:***
Validation message for field left empty:
- Enter a UK mobile number

Validation for all invalid answers, with an example of a valid format (as we do on email):
- Enter a UK mobile number in a valid format, for example, 07700 912345 or +44 7700 912345

The sanitised number will be: 7 xxx xxx xxx
The sanitised number code needs to be common code

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).
